packages/annotations/src/annotations.ts

```diff
@@ -62,28 +62,52 @@ export function getAnnotationServiceUrl(
 export function searchAnnotations(
   requestOptions: IQueryFeaturesRequestOptions
 ): Promise<any /* make it a type */> {
```

Comment:
I think we should export an `IResourceObject` interface as per: http://jsonapi.org/format/#document-resource-objects, and a this should be `Promise<{data: IResourceObject[], included?: IResourceObject[]}`.